test/DePayRouterV1Uniswap01.spec.ts

```diff
@@ -120,7 +120,8 @@ describe('DePayRouterV1 + DePayRouterV1Uniswap01', () => {
         plugins: [uniswapPlugin.address, paymentPlugin.address]
       })
     ).to.be.revertedWith(
-      'UniswapV2Router: EXPIRED'
+      // looks so weird because how the error is forwarded (through delegate call)
```

Comment:
It would be an error of hard hat or EVM package.

Reply:
It has something to do with how we re-throw errors from delegateCall. See here: https://github.com/DePayFi/depay-ethereum-router/blob/master/contracts/DePayRouterV1.sol#L103

Reply:
and waffle/matcha might fix that internally at some point to that we can do regexp match again, until it has to stay so weird.